### DISCUSSION TITLE
直播间自动佩戴勋章功能的默认勋章是怎么决定的

### DISCUSSION THREAD

**Discussion by u2shana** [the1812/Bilibili-Evolved#4350] [提问 / Q&A]
现在的逻辑，进入一个没有粉丝勋章的直播间，就会自动切换成某个默认勋章

但是这个默认勋章是如何选取的呢，也明显不是按照等级、名称、获取时间条件来选择

后续这种情况是否能够添加功能自定义默认勋章/不佩戴勋章呢

**Comment by the1812** [comment]
是在功能里最后一次选中的勋章
